<launch>
	<node pkg = "kalman_filter" type = "kalman_filter_node" name = "kalman_filter" output = "screen">
	
	<param name = "front_sonar_topic" 			type = "string"      value = "/range_front"/>   
	<param name = "left_sonar_topic" 			type = "string"      value = "/range_front_left"/>
	<param name = "right_sonar_topic" 			type = "string"      value = "/range_front_right"/>  
	<param name = "sonar_cmd_vel_topic" 		type = "string"      value = "/cmd_vel/sonar"/> 
	
	</node>
</launch>
